src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html

```diff
@@ -325,7 +325,7 @@ <h3>Court Appearance</h3>
           </span>
         </ng-container>
         <form [formGroup]="courtAppearanceForm" novalidate>
-          <div style="display: grid; grid-template-columns: auto auto auto auto auto auto auto auto auto auto auto">
+          <div style="display: grid; grid-template-columns: 11% 6% 6% 11% 6% 11% 11% 11% 10% 10% 7%">
```

Comment:
Ouch. Can we use margins/padding instead?

Reply:
This is for the width of columns so that it can be responsive

Reply:
It is fine

Reply:
okay